tools/pythonpkg/tests/fast/test_expression.py

```diff
@@ -0,0 +1,557 @@
+
+        # Greater than
+        expr1 = col1 > col2
+        expr2 = col2 > col1
+        expr3 = col3 > col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(False, True, False)]
+
+        # Greater than or equal
+        expr1 = col1 >= col2
+        expr2 = col2 >= col1
+        expr3 = col3 >= col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(False, True, True)]
+
+        # Less than
+        expr1 = col1 < col2
+        expr2 = col2 < col1
+        expr3 = col3 < col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(True, False, False)]
+
+        # Less than or equal
+        expr1 = col1 <= col2
+        expr2 = col2 <= col1
+        expr3 = col3 <= col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(True, False, True)]
+
+    def test_expression_alias(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+			select 1 as a
+		"""
+        )
+        col = ColumnExpression('a')
+        col = col.alias('b')
+
+        rel2 = rel.select(col)
+        assert rel2.columns == ['b']
+
+    def test_star_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+			select
+				1 as a,
+				2 as b
+		"""
+        )
+        star = StarExpression()
+        rel2 = rel.select(star)
+        res = rel2.fetchall()
+        assert res == [(1, 2)]
+
+        # With exclude list
+        star = StarExpression(exclude=['a'])
+        rel2 = rel.select(star)
+        res = rel2.fetchall()
+        assert res == [(2,)]
+
+    def test_struct_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+			select
+				1 as a,
+				2 as b
+		"""
+        )
+
+        col1 = ColumnExpression('a')
+        col2 = ColumnExpression('b')
+        expr = FunctionExpression('struct_pack', col1, col2).alias('struct')
+
+        rel = rel.select(expr)
+        res = rel.fetchall()
+        assert res == [({'a': 1, 'b': 2},)]
+
+    def test_function_expression(self):
+        con = duckdb.connect()
+
+        def my_simple_func(a: int, b: int, c: int) -> int:
+            return a + b + c
+
+        con.create_function('my_func', my_simple_func)
+
+        rel = con.sql(
+            """
+			select
+				1 as a,
+				2 as b,
+				3 as c
+		"""
+        )
+        col1 = ColumnExpression('a')
+        col2 = ColumnExpression('b')
+        col3 = ColumnExpression('c')
+        expr = FunctionExpression('my_func', col1, col2, col3)
+        rel2 = rel.select(expr)
+        res = rel2.fetchall()
+        assert res == [(6,)]
+
+    def test_case_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+			select
+				1 as a,
+				2 as b,
+				3 as c,
+		"""
+        )
+
+        col1 = ColumnExpression('a')
```

Comment:
Any tests where this Column expression doesn't exist?

Reply:
in `test_column_expression`
```py
        column = ColumnExpression('d')
        with pytest.raises(duckdb.BinderException, match='Referenced column "d" not found'):
            rel2 = rel.select(column)
```